_labs/multimeters_and_measurements.md

```diff
@@ -5,89 +5,99 @@ layout: lab
 ---
 
 ## Overview
-In this lab we will be using the tools we know and measure DC voltage, DC current, and resistance. You will learn how to operate circuits using breadboards and how to use multimeters.  
+In this lab we will be using the tools we know and measure DC voltage, DC current, and resistance. You will learn how to operate circuits using breadboards and how to use multi-meters.  
```

Comment:
You changed this from *multimeters* to *multi-meters*, but I think *multimeters* is the correct one to use.